packages/debug/src/browser/style/index.css

```diff
@@ -401,15 +401,21 @@
     border-bottom-color: var(--theia-debugExceptionWidget-border);
     background-color: var(--theia-debugExceptionWidget-background);
     padding: var(--theia-ui-padding) calc(var(--theia-ui-padding)*1.5);
-	white-space: pre-wrap;
-	user-select: text;
+    white-space: pre-wrap;
+    user-select: text;
     overflow: hidden;
 }
 
 .theia-debug-exception-widget .title {
-	font-weight: bold;
+    font-weight: bold;
```

Comment:
Minor: I noticed that we use a mono-spaced font for the title, while vscode uses a normal text font:

vscode:

![image](https://user-images.githubusercontent.com/4377073/179964561-5ca0e75c-7f7b-4e85-b535-3f0aa7630ee7.png)

theia:

![image](https://user-images.githubusercontent.com/4377073/179964763-12f7b6b9-2364-4749-a57e-fa7b8c386846.png)